src/libaudqt/prefs-window-qt.cc

```diff
@@ -557,12 +557,16 @@ static void create_plugin_category (QStackedWidget * parent)
 
     s_plugin_view->setModel (s_plugin_model);
     s_plugin_view->setSelectionMode (QTreeView::NoSelection);
+    s_plugin_view->setFont (get_font_for_class ("QListView"));
+    s_plugin_view->setAlternatingRowColors (true);
 
     auto header = s_plugin_view->header ();
 
     header->hide ();
     header->setSectionResizeMode (header->ResizeToContents);
-    header->setStretchLastSection (false);
+    // stretch the last section so the QTreeView takes the entire
+    // width of its parent QStackedWidget.
+    header->setStretchLastSection (true);
```

Comment:
This causes trouble since now clicking anywhere to the right of the "settings" icon opens the settings for that plugin.